app/assets/stylesheets/application.css

```diff
@@ -10,8 +10,41 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
- *= require_self
- *= require foundation_and_overrides
+*= require foundation_and_overrides
+*= require_tree .
+*= require_self
 
  */
+body {
+    background-color: #98C1D9;
+}
+
+.top-bar {
+    background-color: #EE6C4D;
+    color: #E0FBFC;
+    font-weight: bold;
+    border-bottom: 5px solid #293241;
+}
+
+.top-bar-section p {
+    margin: 2px;
+}
+
+.top-bar a {
+    color:#293241;
+    font-weight: bolder;
+}
+
+.top-bar h1 {
+    font-family: 'Play', sans-serif;
+    margin-left: 20px;
+}
+
+.sticky {
+    width: 100%;
+}
+
+.content {
+    margin: 2em;
+    margin-top: 3em;
+}
```

Comment:
yay styling! these should be removed from this manifest file and put in their own stylesheet like `main.css`

Reply:
Moved into `main.scss` per suggestion